styles.css

```diff
@@ -525,3 +551,308 @@ form {
   background: #0d151e;
   border-radius: 10px;
 }
+
+#footer-contact {
+  display: flex;
+  margin-top: 10px;
+  list-style-type: none;
+  justify-content: center;
+}
+
+#footer-contact > li {
+  margin-right: 10%;
+}
+
+.form-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .right-side-image {
+    display: none;
+  }
```

Comment:
- [ ] I observed that you repeated this line of code a lot, this is not the best according to CSS best practices. Kindly implement it the way you did in line 189 -190 for all the repeated code. 👍
Note: Please kindly look through lines (596, 691, 708, 743, 777, 797,815 and 851) of code carefully. 👍